app/main.cpp

```diff
@@ -43,13 +44,17 @@
 #include "nucleus/map_label/CameraTransformationProxyModel.h"
 #include "nucleus/map_label/MapLabelModel.h"
 
+#include "nucleus/camera/PositionStorage.h"
+#include "nucleus/utils/UrlModifier.h"
+
 int main(int argc, char **argv)
 {
     //    QGuiApplication::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::Floor);
     QQuickWindow::setGraphicsApi(QSGRendererInterface::GraphicsApi::OpenGLRhi);
-    QGuiApplication app(argc, argv);
+    QApplication app(argc, argv);
+    app.setWindowIcon(QIcon("app/icons/favicon.ico"));
     QCoreApplication::setOrganizationName("AlpineMaps.org");
-    QCoreApplication::setApplicationName("AlpineApp");
+    QCoreApplication::setApplicationName("Alpine Maps");
```

Comment:
Again QGuiApplication not possible because we need support for QWidget-based classes for the charts.

Reply:
applicationDisplayName is also in qapplication due to inheritance: https://doc.qt.io/qt-6/qapplication-members.html

why not change it? because application name is used in configuration paths, cache paths etc. 1. spaces could cause issues, 2. it would reset the cache in existing installations (i.e., manu's and my phone).